### PR TITLE
909942 - Updated the Getting-Started page for Tab view, Navigation drawer, Busy indicator and effects view controls

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Updated the Getting-Started page for Tab view, Navigation drawer, Busy indicator and effects view controls

### CODE REVIEW

File: MAUI/NavigationDrawer/Getting-Started.md [hunk 1]
```diff
@@ -7,17 +7,32 @@ control: NavigationDrawer
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Navigation Drawer (SfNavigationDrawer)
+# Getting Started with .NET MAUI Navigation Drawer
 
-This section explains how to add the [.NET MAUI NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) control. This section covers only the basic features needed to get started with Syncfusion .NET MAUI NavigationDrawer.
+This section guides you through setting up and configuring a [NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) in your .NET MAUI application. Follow the steps below to add a basic Tab View to your project.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Getting-Started.md:12]: This section guides you through setting up and configuring a [NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) in your .NET MAUI application. Follow the steps below to add a basic Navigation Drawer to your project.

File: MAUI/NavigationDrawer/Getting-Started.md [hunk 2]
```diff
@@ -49,54 +64,46 @@ namespace NavigationDrawerSample
...
 
 {% tabs %}
 
 {% highlight xaml %}
 
-
-<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+<ContentPage
+    . . .    
+    xmlns:navigationdrawer="clr-namespace:Syncfusion.Maui.NavigationDrawer;assembly=Syncfusion.Maui.NavigationDrawer">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Getting-Started.md:79]: xmlns:navigationDrawer="clr-namespace:Syncfusion.Maui.NavigationDrawer;assembly=Syncfusion.Maui.NavigationDrawer">

File: MAUI/TabView/Getting-Started.md [hunk 3]
```diff
@@ -7,21 +7,36 @@ control: Tab View
...
+This section guides you through setting up and configuring a [Tab View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html?tabs=tabid-1) in your .NET MAUI application. Follow the steps below to add a basic Tab View to your project.
+
+## Prerequisites
+
+Before proceeding, ensure the following are set up:
+
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
 
 To get start quickly with our .NET MAUI Tab View, you can check the below video.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Getting-Started.md:21]: move the video link first, and then add prerequisites

File: MAUI/NavigationDrawer/Getting-Started.md (hunk 1 above)
Comment (Arunachalam3641) [reply MAUI/NavigationDrawer/Getting-Started.md:12]: Yes, I have changed

File: MAUI/NavigationDrawer/Getting-Started.md (hunk 2 above)
Comment (Arunachalam3641) [reply MAUI/NavigationDrawer/Getting-Started.md:79]: Yes, I have changed

File: MAUI/NavigationDrawer/Getting-Started.md [hunk 4]
```diff
@@ -152,7 +159,7 @@ N> To change the side of the navigation pane, utilize the [Position](https://hel
...
+Create an ImageButton and set the required image to the `Source` property. Add the image to the `Resources/Images` folder. Subscribe Clicked event of the button and invoke the [ToggleDrawer()](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html#Syncfusion_Maui_NavigationDrawer_SfNavigationDrawer_ToggleDrawer) method to toggle the drawer. Properly align the layout of [ContentView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html#Syncfusion_Maui_NavigationDrawer_SfNavigationDrawer_ContentView) to position the hamburger icon at the top left, as demonstrated in the following code.
```
[the remark this answers is not in the document]
Comment (Arunachalam3641) [reply MAUI/NavigationDrawer/Getting-Started.md:162]: yes, I have changed

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 5]
```diff
@@ -8,23 +8,39 @@ documentation: ug
 keywords : .net maui busy indicator, maui busy indicator, .net maui activity indicator, maui activity indicator, .net maui loading indicator, maui loading indicator.
 ---
 
-# Getting Started with .NET MAUI Busy Indicator (SfBusyIndicator)
+# Getting Started with .NET MAUI Busy Indicator
 
-## Getting started with .NET MAUI Busy Indicator
+This section guides you through setting up and configuring a
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Getting-Started.md:13]: This section guides you through setting up and configuring a [Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1)
in your .NET MAUI application. Follow the steps below to add a basic Busy Indicator to your project.

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 6]
```diff
@@ -8,23 +8,39 @@ documentation: ug
...
-Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
 
-## Handler registration 
+1.  In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
+3.  Ensure the necessary dependencies are installed correctly, and the project is restored.
 
- In the MauiProgram.cs file, register the handler for Syncfusion core.
+## Step 3: Handler registration
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Getting-Started.md:41]: ## Step 3: Register the handler

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 7]
```diff
@@ -8,23 +8,39 @@ documentation: ug
...
 
-## Handler registration 
+1.  In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
+3.  Ensure the necessary dependencies are installed correctly, and the project is restored.
 
- In the MauiProgram.cs file, register the handler for Syncfusion core.
+## Step 3: Handler registration 
+
+In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Getting-Started.md:43]: Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 8]
```diff
@@ -56,57 +72,53 @@ namespace BusyIndicatorSample
...
-
-
-{% endhighlight %}
+using Syncfusion.Maui.Core;
 
-{% highlight c# %}
-          
-SfBusyIndicator busyIndicator = new SfBusyIndicator(){IsRunning = true}; 
-Content = busyIndicator;  
+namespace BusyIndicatorMauiSample
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Getting-Started.md:102]: use same namespace in all the code snippets

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 9]
```diff
@@ -56,57 +72,53 @@ namespace BusyIndicatorSample
...
+		}  
+	}  
+}  
 
 {% endhighlight %}
 
 {% endtabs %}
 
-## Setting Animation Type in .NET MAUI Busy Indicator
+## Step 5: Customizing the Busy Indicator
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Getting-Started.md:121]: Step 5: Setting Animation Type in .NET MAUI Busy Indicator

File: MAUI/Effects-View/Getting-Started.md [hunk 10]
```diff
@@ -7,19 +7,35 @@ control: Effects View
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Effects View (SfEffectsView)
+# Getting Started with .NET MAUI Effects View
+This section guides you through setting up and configuring a
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Effects-View/Getting-Started.md:11]: This section guides you through setting up and configuring a [Effects View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html?tabs=tabid-1)
in your .NET MAUI application. Follow the steps below to add [Effects View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html?tabs=tabid-1) to your project.

File: MAUI/Effects-View/Getting-Started.md [hunk 11]
```diff
@@ -7,19 +7,35 @@ control: Effects View
...
+3. Name the project and choose a location, then click Create.
 
-## Handler registration 
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
+
+1.  In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
+3.  Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Handler registration
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Effects-View/Getting-Started.md:38]: ## Step 3: Register the handler

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 5 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:13]: Yes, I have changed.

File: MAUI/Effects-View/Getting-Started.md [hunk 12]
```diff
@@ -7,19 +7,35 @@ control: Effects View
...
-## Handler registration 
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
+
+1.  In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
+3.  Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Handler registration 
 
  In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Effects-View/Getting-Started.md:40]: Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 6 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:41]: Yes, I have changed.

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 7 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:43]: Yes, I have changed.

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 13]
```diff
@@ -56,57 +72,53 @@ namespace BusyIndicatorSample
...
+## Step 4: Add a Basic Busy Indicator
 
-### Adding the .NET MAUI Busy Indicator control
+1. To initialize the control, import the Core namespace into your code.
 
-Step 1: Add the NuGet to the project as discussed in the above reference section. 
+2. Initialize [SfBusyIndicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1)
 
-Step 2: Add the namespace as shown in the following code sample, to initialize the .NET MAUI Busy Indicator.
+Add the namespace as shown in the following code sample.
```
[the remark this answers is not in the document]
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:81]: Yes, I have removed.

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 8 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:102]: Yes, I have changed.

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 9 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Getting-Started.md:121]: Yes, I have changed.

File: MAUI/Effects-View/Getting-Started.md (hunk 10 above)
Comment (Arunachalam3641) [reply MAUI/Effects-View/Getting-Started.md:11]: Yes, I have changed.

File: MAUI/Effects-View/Getting-Started.md (hunk 11 above)
Comment (Arunachalam3641) [reply MAUI/Effects-View/Getting-Started.md:38]: Yes, I have changed.

File: MAUI/Effects-View/Getting-Started.md (hunk 12 above)
Comment (Arunachalam3641) [reply MAUI/Effects-View/Getting-Started.md:40]: Yes, I have changed.

File: MAUI/Avatar-view/Getting-Started.md [hunk 14]
```diff
@@ -11,31 +11,40 @@ documentation: ug
...
-{% youtube "https://www.youtube.com/watch?v=XrTvIgeeUm0" %}
+### Visual Studio Code
 
-## Step 1: Create a New MAUI Project
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
+
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Avatar-view/Getting-Started.md:33]: Step 2 showing 2 times

File: MAUI/NavigationDrawer/Getting-Started.md [hunk 15]
```diff
@@ -7,17 +7,45 @@ control: NavigationDrawer
...
+## Step 1: Create a New MAUI Project
+
+### Visual Studio Code
+
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
+
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Getting-Started.md:34]: Install the Syncfusion MAUI NavigationDrawer NuGet Package

File: MAUI/NavigationDrawer/Getting-Started.md [hunk 16]
```diff
@@ -7,17 +7,45 @@ control: NavigationDrawer
...
+
+1.  In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NavigationDrawer/Getting-Started.md:37]: Syncfusion MAUI NavigationDrawer

**Review by PaulAndersonS — APPROVED** [review]

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**